dronegen/common.go

```diff
@@ -265,13 +265,19 @@ func releaseMakefileTarget(b buildType) string {
 		makefileTarget += "-fips"
 	}
 
-	// Override Windows targets.
+	// Override non-Unix targets.
```

Comment:
Darwin is a Unix target, no?

Reply:
Sorry yes that is poorly worded - will fix after I do some more testing